README.md

```diff
@@ -2,38 +2,43 @@
   <img src="https://avatars0.githubusercontent.com/u/41934032?s=400&v=4" alt="Crypto.com Chain" width="400">
 </p>
 
-# Crypto.com Chain Sample Wallet
+## Crypto.com Chain Sample Wallet
 
 This repository contains a sample implementation of Crypto.com Chain wallet with integration of [client JSON-RPC server](https://github.com/crypto-com/chain/tree/master/client-rpc).
 
-## Contribution
-
-TODO
```

Comment:
contribution guidelines are still TODO -- since this is JS sample code, its guidelines can differ from the main chain repo

Reply:
Added contribution section back